src/js/menu/acquia_lift.personalize.views.js

```diff
@@ -934,37 +635,19 @@
       if (!this.model) {
         return;
       }
-      if (this.model instanceof Drupal.acquiaLiftUI.MenuCampaignABModel) {
-        // Simple A/B campaigns need to call the executor for each of the
-        // options within the selected page variation.
-        var variation_index = this.model.get('activeVariation');
-        var variations = this.model.get('optionSets').getVariations();
-        var variation = _.find(variations, function(obj) {
-          return obj.original_index == variation_index;
-        });
-
-        if (!variation) return;
-        var i, num = variation.options.length, current;
-        // Run the executor for each option in the variation.
-        for (i=0; i < num; i++) {
-          current = variation.options[i];
-          Drupal.personalize.executors[current.executor].execute($(current.selector), current.option.option_id, current.osid);
+      // Standard tests just call the executors on the selected option.
```

Comment:
This line is no longer valid since "standard" tests compared to what?  I think leaving the second two lines can still be helpful to remember what the model is representing in the callback.